### PR TITLE
Yandex Id System: refactoring

### PR DESCRIPTION
**Pull request by someden**
## Type of change

- [x] Refactoring (no functional changes, no api changes)

## Description of change
Added some additional logs in Yandex Id System.
Improved some jsdoc.

### CODE REVIEW

**Review by chernodub — APPROVED** [review]

**Comment by someden** [comment]
@smenzer could you please take a look?

File: src/storageManager.js
Comment (smenzer) [inline src/storageManager.js:1]: @ChrisHuie are you ok with the changes to this file, since it's a common file, but only changes were in the comments.

**Review by smenzer — APPROVED** [review]
overall looks fine, just one question for the prebid team since you modified a common file and that normally requires a second review before merging. Once Chris confirms on that file, the rest of this looks fine and can be merged.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**
